sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -120,31 +146,112 @@
 
   @Override
   public Span.Builder setRecordEvents(boolean recordEvents) {
-    this.recordEvents = recordEvents;
```

Comment:
Why removing this? My suggestion was that you can avoid using `Boolean` for `this.recordEvents` and use a `boolean`. The difference is that you cannot have `null` or unset, and I suggested that the default value for `this.recordEvents` should be `false`.

Reply:
BC I cannot use it and the build fails on waring - unused variable. 

I wanted to add it as part of https://github.com/open-telemetry/opentelemetry-java/issues/438.


I have added it back with `@SuppressWarnings` and link to the issue.